tests/autoyast/installation.pm

```diff
@@ -375,10 +376,13 @@ sub run {
         elsif (match_has_tag('lang_and_keyboard')) {
             return;
         }
-        elsif (match_has_tag('opensuse-welcome')) {
-            return;             # Popup itself is processed in opensuse_welcome module
+        elsif (match_has_tag('opensuse-welcome') || match_has_tag('opensuse-welcome-started-not-shown-bsc1169203')) {
+            untick_welcome_on_next_startup_and_close() if match_has_tag('opensuse-welcome');
```

Comment:
I have moved processing of that screen to separate module which serves this issue as this module became monster already and does everything. So I would propose to have those workaround in `installation/opensuse_welcome`

Do we see this issue with non AY installations?

Reply:
opensuse_welcome is wrong in the majority of scenarios, like this one, the popup has to be handled in the module where the first login appears, in this case, installation.

Reply:
We don't login in the installation and as per your own words, login is not part of it. So I believe this approach of handling everything in single place is wrong and hard to maintain with tons of those if statements.

Reply:
It is more unmaintainable to spread the if in any following module (to blindly accept the popup as a valid `generic-desktop` until you get to **opensuse_welcome** module, where each module in between can face that bug.
The problem is complex, and the design of the module **installation** asks for handling everything in one place. The product doesn't behave like our test modules, our test modules should behave after the product's behavior.

Would you like to split the module
- installation
to
- installation (stops before rebooting)
- reboot_after_installation (accepts the reboot and stops at GRUB, ensuring no timeout)
- grub_test_post_install_config (handles GRUB, wait for config to finish and stops at GDM[bsc1169203] or generic-desktop)
?

Then we will no have a module monster.